src/within.ts

```diff
@@ -1,11 +1,16 @@
 import type { ReactTestInstance } from 'react-test-renderer';
-import { a11yAPI } from './queries/a11yAPI';
 import { bindByTextQueries } from './queries/text';
 import { bindByTestIdQueries } from './queries/testId';
 import { bindByDisplayValueQueries } from './queries/displayValue';
 import { bindByPlaceholderTextQueries } from './queries/placeholderText';
 import { bindUnsafeByTypeQueries } from './queries/unsafeType';
 import { bindUnsafeByPropsQueries } from './queries/unsafeProps';
+import { bindByLabelTextQueries } from './queries/labelText';
```

Comment:
nit: Can we move these imports above `unsafe` and have them sorted: label, role, a11yhint, a11y state, a11y states, a11y value?

Reply:
done